addons/storyshots/storyshots-puppeteer/src/imageSnapshot.ts

```diff
@@ -11,7 +11,6 @@ export const imageSnapshot = (customConfig: Partial<ImageSnapshotConfig> = {}) =
   return puppeteerTest({
     ...config,
     async testBody(page, options) {
-      expect.assertions(1);
```

Comment:
WDYT about `expect.hasAssertions()` instead? it seems like there's already an assertion being made, and this would ensure that there >= 1 assertion.

Reply:
That's good idea!